src/components/Link/Link.jsx

```diff
@@ -29,13 +31,16 @@ Link.propTypes = {
   className: PropTypes.string,
   /** Title of the link. */
   title: PropTypes.string,
+  /** Font weight of the link. */
+  fontWeight: PropTypes.oneOf([500, 700]),
```

Comment:
Tried with `PropTypes.oneOf([fontWeightMedium, fontWeightBold]),` but for some reason that breaks the snapshot testing.